service/src/main/java/gov/nasa/pds/api/engineering/elasticsearch/business/RequestAndResponseContext.java

```diff
@@ -161,10 +161,17 @@ private RequestAndResponseContext(
 		}
 		else
 		{
+			String known = "";
 			log.warn("Could not find a matach for application type: " + String.valueOf(this.format));
 			log.warn("   Known types: " + String.valueOf(this.formatters.keySet().size()));
-			for (String key : this.formatters.keySet()) log.warn("      key: " + String.valueOf(key));
-			throw new ApplicationTypeException("The given application type, " + String.valueOf(this.format) + ", is not known by RquestAndResponseContext.");
+			for (String key : this.formatters.keySet())
+			{
+				log.warn("      key: " + String.valueOf(key));
+				known = known + key + ",";
+			}
+			known = known.substring(0, known.length()-1);
+			throw new ApplicationTypeException("The given application type, " + String.valueOf(this.format) +
```

Comment:
I would rephrase the message like:

"The Accept header value " + String.valueOf(this.format) + " is not supported, supported values are " + known

Maybe that sentence does not make sense in the context of the exception but we need a message which makes sense to the user of the API, so that he understands what he need to change in his request.

Reply:
resolved